cdap-ui/app/cdap/text/text-en.yaml

```diff
@@ -148,18 +148,6 @@ features:
         label: "System Compute Profiles "
         labelWithCount: "System Compute Profiles ({count})"
         import: Import
-    Component-Overview:
-      emptyMessage: Available only in Distributed mode
-      headers:
```

Comment:
I don't think we should still remove this @tonybach .
<img width="1680" alt="screen shot 2018-05-07 at 11 42 09 am" src="https://user-images.githubusercontent.com/1452845/39718627-b431e490-51eb-11e8-99b6-402888d1bb7e.png">